tests/rptest/tests/leadership_transfer_test.py

```diff
@@ -333,9 +333,15 @@ def all_partitions_present(num_nodes, per_node=None):
                    err_msg="Leadership did not stablize")
 
         shard2leaders = self._get_leaders_by_shard()
-        self.redpanda.logger.debug(f"Leaders by shard: {shard2leaders}")
-        expected_on_shard = sum(shard2leaders.values()) / len(shard2leaders)
+        expected_on_shard = sum(shard2leaders.values()) / (
+            len(self.redpanda.nodes) * self.redpanda.get_node_cpu_count())
+        self.redpanda.logger.info(
+            f"Leaders by shard: avg: {expected_on_shard}, min: {min(shard2leaders.values())}"
+            f", full stats: {shard2leaders}")
         for s, count in shard2leaders.items():
-            expected_min = math.floor(expected_on_shard * 0.8)
+            # Check with a lot of slack because leader balancer may not be able to achieve
```

Comment:
wonder if we should mark this as ok_to_fail instead, so we don't lose track of tightening the check once the underlying issue is fixed.

Reply:
Well if it is marked ok_to_fail we will surely lose track :) and I don't think we can mark individual assertions ok_to_fail... Also even in this form the check is somewhat useful